src/message/message.go

```diff
@@ -67,7 +67,7 @@ func paragraph(format string, a ...any) string {
 }
 
 func debugPrinter(offset int, a ...any) {
-	printer := pterm.Debug.WithShowLineNumber(logLevel > 2).WithLineNumberOffset(offset)
+	printer := pterm.Debug.WithShowLineNumber(logLevel <= TraceLevel).WithLineNumberOffset(offset)
 	now := time.Now().Format(time.RFC3339)
 	// prepend to a
 	a = append([]any{now, " - "}, a...)
```

Comment:
pterm.error down on `89` should also be logLevel <= TraceLevel

Reply:
👍 I didn't see that, thanks. Updated in de6040d